xdrip/Managers/LibreLinkUp/LibreLinkUpFollowManager.swift

```diff
@@ -572,7 +572,8 @@ class LibreLinkUpFollowManager: NSObject {
         // this is pretty much the same request as done in requestLogin()
         var request = URLRequest(url: url)
         request.setValue("Bearer \(token)", forHTTPHeaderField: "Authorization")
-        
+        request.setValue((libreLinkUpPatientId ?? libreLinkUpId ?? "").sha256(), forHTTPHeaderField: "Account-Id")
```

Comment:
If this header is only required for version 4.11.0+, maybe we should condition sending it to those versions as well.

Reply:
Very probably in the upcoming days they will require the newer version number like they did  at once when they released LibreLinkUp 4.7.

Luckily it was just an one-liner fix (https://github.com/creepymonster/GlucoseDirect/commit/3667f439de8f8110c18856222f744fb614a03405#diff-64a6a7ce7140299dd152ac5571715c438fa00b102ef9ec7e89328c25a0fcc991L166) so hopefully our apps will survive while the external WebFollowers of Android’s xDrip, Sweet Dreams and Gluroo will be blocked until they cache each user's credentials... ;-)

Anyway the additional header doesn't harm when sent with the current default 4.7.0 version number.